benchopt/tests/__init__.py

```diff
@@ -53,7 +53,13 @@ def __exit__(self, exc_class, value, traceback):
         )
         if len(self.result_files) >= 1:
             for result_file in self.result_files:
-                Path(result_file).unlink()
+                path_ = Path(result_file)
+                path_.unlink()  # remove csv file
+                dir_ = path_.parents[0]
+                stem = path_.stem
+                html_files = dir_.glob('*' + stem + '*.html')
+                for html_file in html_files:
+                    html_file.unlink(missing_ok=True)  # remove html files
```

Comment:
Some variable renaming, but this is completely nitpicking :sweat_smile: 

```suggestion
                result_path = Path(result_file)
                result_path.unlink()  # remove csv file
                result_dir = result_path.parents[0]
                stem = result_path.stem
                for html_file in result_dir.glob(f'*{stem}*.html'):
                     # remove html files associated with this results
                    html_file.unlink(missing_ok=True)
```